backend/src/main/java/com/hyooun/wilson/review/adapter/out/persistence/ReviewJpaEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.hyooun.wilson.review.adapter.out.persistence;
+
+import com.hyooun.wilson.review.domain.Change;
+import com.hyooun.wilson.review.domain.ReviewResult;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Entity
+@Getter
```

Comment:
ReviewJpaEntity 클래스에 @AllArgsConstructor 애너테이션을 사용하여 생성자를 추가하세요.

```suggestion
@AllArgsConstructor
```